exe/genv-envs

```diff
@@ -153,7 +220,15 @@ def do_ps(envs: Envs, format: str, header: bool, timestamp: bool) -> None:
         elif format == 'tui':
             print(f'{eid:<8}{user:<16}{name:<16}{created:<21}{pids}')
 
+
 def do_query(envs: Envs, eid: Optional[str], queries: List[str]) -> None:
+    """
+    Query environment data using a query string. The queries results will be printed into the stdio.
+    :param envs: dictionary describing all existing environments.
+    :param eid: eid pointing to the environment to query. If none, query your current environment.
```

Comment:
if `None` it queries all environments

Reply:
My mistake. Fixed to :
"If none, query all environments"